src/helpers/installer.ts

```diff
@@ -0,0 +1,102 @@
+import puppeteer, { BrowserFetcher, RevisionInfo } from 'puppeteer-core';
+import ProgressBar from 'progress';
+import preLogger from './logger';
+
+let browserFetcher: BrowserFetcher;
+
+// Override current environment proxy settings with npm configuration, if any.
+const NPM_HTTPS_PROXY =
+  process.env.npm_config_https_proxy || process.env.npm_config_proxy;
+const NPM_HTTP_PROXY =
+  process.env.npm_config_http_proxy || process.env.npm_config_proxy;
+const NPM_NO_PROXY = process.env.npm_config_no_proxy;
+
+if (NPM_HTTPS_PROXY) process.env.HTTPS_PROXY = NPM_HTTPS_PROXY;
```

Comment:
We could use if statements with paranthesis and curly bracets to make the app written in a consistent way but It doesn't effect the approval for now. We can consider it to discuss in future.

Reply:
Current eslint and prettier configurations allow this style. They are put in to avoid styling discussions 🙂